ee/tabby-schema/src/schema/repository/mod.rs

```diff
@@ -260,7 +260,7 @@ pub trait RepositoryProvider {
 }
 
 #[async_trait]
-pub trait RepositoryService: Send + Sync + RepositoryAccess {
```

Comment:
Reason for not keeping RepositoryService as a supertrait of ConfigAccess?

Reply:
methods like `list_crawl_urls` no longer fit scope of `RepositoryService`